packages/longhorn/package.yaml

```diff
@@ -1,7 +1,7 @@
 url: https://github.com/longhorn/charts.git
 subdirectory: charts/longhorn
-commit: 299f4ab8ad01137c463574fe013da30efc6b05d0
-packageVersion: 01
+commit: 93f250e4a175044d34c6982ea6168b213caeee9f
```

Comment:
Use `commit: v1.1.1` would be easier for future maintenance.

Reply:
Though unusually, tag can be re-tag to a different commit hash and content.  This might cause some confusion.

Reply:
Ideally, the tag is immutable from the perspective of project management. You don't need to worry about that part.

Reply:
Our tooling doesn't support tag yet, just sha.

Reply:
In Git, the tag implies the SHA commit ID.
I tried to use the tag, it works the same as using SHA commit ID.